content/workload-reduction-toolkit/address-workload-issues/feedback-and-marking/review-and-streamline-feedback-and-marking.html.erb

```diff
@@ -0,0 +1,137 @@
+---
+title: Review and streamline feedback and marking
+---
+
+<div class="govuk-main-wrapper">
+  <div class="dfe-width-container govuk-grid-row two-column-page">
+    <div class="govuk-grid-column-two-thirds">
+      <div class="dfe-width-container govuk-grid-row">
+        <h1 class="govuk-heading-l govuk-!-margin-bottom-3">Review and streamline feedback and marking</h1>
+        <div class="dfe-width-container govuk-grid-row govuk-!-margin-bottom-7">
+          <strong class="govuk-tag">Presentation</strong>
+        </div>
+        <div class="dfe-width-container govuk-grid-row">
+          <p>
+            Use this presentation in a staff meeting or INSET day to review and streamline your feedback and marking processes. 
+          </p>
+          <p>
+            The presentation includes:
+          <ul>
+            <li>background and contextual information</li>
+            <li>the purposes of feedback and marking</li>
+            <li>reviewing current practice</li>
+            <li>next steps</li>
+          </ul>
+          </p>
+        </div>
+      </div>
+
+      <div class="dfe-width-container govuk-grid-row">
+        <ul class="resource-card-group govuk-!-margin-top-8 govuk-!-margin-bottom-0">
```

Comment:
I realise this is copied from the presentation page, but I've just realised this html and css will need changing. It shouldn't be a `<ul>` since it's not a list (bad for accessibility) and it shouldn't really be using `resource-card-group` since it's not a resource card. I can address this in my current 'Staff workload survey' PR since it will use the same code.

Reply:
Abi is resolving this in a separate PR